lib/plugins/unthreaded-signing/plugin.c

```diff
@@ -36,35 +37,81 @@
 #endif
 #endif
 
+/**
+ * Structure to store the signature information.
+ */
+typedef struct _sv_unthreaded_signature_info_t {
+  uint8_t *signature;  // The signature of the |hash|.
+  size_t signature_size;  // The size of the |signature|.
+  size_t max_signature_size;  // The allocated size of the |signature|.
+} sv_unthreaded_signature_info_t;
```

Comment:
This is not a public struct, hence keep the name shorter, for example `signature_data_t`